### ISSUE TITLE
Rebuttal to commit #890e8d4

### ISSUE THREAD

**Issue by AH-dark** [solstice23/argon-theme#561]
### Issue 内容

In the commit https://github.com/solstice23/argon-theme/commit/890e8d445f5980bfcbcc6d6c70811c45a979fd7f by @LiCaoZ, he changed the method of obtaining the avatar from the QQ account: the directly outputted avatar link with the QQ number was changed to obtain the avatar without the QQ number through the API.

I don't think this is a good approach, this calls a lot of HTTP requests in the page rendering.

First, for an HTTP request, it may cause a delay of at least 100ms. 
But here, because all avatars acquisition need to call the API, not only may cause flood, but such a huge num of requests (one page can reach dozens of times) also may be difficult to bear. 
In this case, page rendering time and TTFB will be greatly extended.

In addition, there is no cache in any form for the avatars obtained by the API of the same user (same QQ number), which is obviously inefficient.

Moreover, this API is not a public API, and its stability is not guaranteed.

In conclusion, I suggest @solstice23 to reconsider this commit or review it.

**Comment by solstice23** [comment]
Thanks for your suggestion! I merged that PR too rashly. Now it has been reverted.
